my_game.py

```diff
@@ -352,11 +371,192 @@ def on_mouse_press(self, x: int, y: int, button: int, modifiers: int):
         """
         if arcade.get_distance(x, y, CONFIG['BUTTON_X'], CONFIG['BUTTON_Y']) < self.play_button.width // 2:
             self.start_game()
+            
+        if arcade.get_distance(x, y, CONFIG['SETTINGS_BUTTON_X'], CONFIG['SETTINGS_BUTTON_Y']) < self.settings_button.width // 2:
+            self.enter_settings()
 
     def start_game(self):
         in_game_view = InGameView()
         self.window.show_view(in_game_view)
 
+    def enter_settings(self):
+        settings_view = SettingsView()
+        self.window.show_view(settings_view)
+            
+            
+class SettingsView(arcade.View):
+    """
+    Veiw for the Settings Screen
+    """
+
+    # Making dict for keys and key IDs
```

Comment:
We need some explanation on what you are trying to do here.

Reply:
Ok i'll do that